utilities/ldap_sync.sql

```diff
@@ -1,11 +1,138 @@
-/* 
-The below scripts will help you synchronize database users with LDAP users. The script will search for roles that contain 
+/*
+======================================
+---------      Overview     ----------
+======================================
+The below scripts will help you synchronize database users with LDAP users. This solution will search for roles that contain 
 a distinguished name as a role comment and will then pull the members of this group from LDAP, create the necessary users, 
 and grant permissions. When users are removed from the group in LDAP, the users will also have the role revoked from them
 
 For more information, please see the below community article:
 https://community.exasol.com/t5/database-features/synchronization-of-ldap-active-directory-groups-and-members-to/ta-p/1686
 
+======================================
+--------- Where to begin?   ----------
+======================================
+About:  These are suggestions to ease implementation of the LDAP Sync process.
+
+1) At the top of each script/udf is "open schema xxxxx". Set the schema in this script,
+   GET_AD_ATTRIBUTES and LDAP_HELPER. The default is to create and use the schema "EXA_TOOLBOX".
+   Your use case will determine the appropriate schema. This is important, as we have 
+   remove the hardcoded schemas in each of the scripts provided, as they previously were using 
+   the default schema "EXA_TOOLBOX".
```

Comment:
EXA_TOOLBOX could still be found in the comments and script bodies.
I'm not sure that it is error, even taking into account this overview sentence.
Just wanted to point to this fact.